torch/quantization/_quantize_script.py

```diff
@@ -62,6 +62,16 @@ def __setstate__(self, state):
     linear_packed_params = torch.jit.script(torch.nn.quantized.modules.linear.LinearPackedParams())._c
     conv_packed_params = torch.jit.script(ConvPackedParams())._c
 
+def _preprocess_qconfig_dict_helper(module, module_types, qconfig_dict, prefix=''):
+    dict_w_names = {}
+    for name, child in module.named_children():
+        module_prefix = prefix + '.' + name if prefix else name
+        if type(child) in module_types:
```

Comment:
why do you need to get module types? can't just look up this in qconfig_dict?

Reply:
Yeah, will fix this